hrjobcontract/js/test/controllers/modal/modal-contract.controller.spec.js

```diff
@@ -322,6 +334,53 @@ define([
       });
     });
 
+    describe('Saving the change revision', function () {
+      beforeEach(function () {
+        mockUIBModal({
+          ModalDialogController: 'change',
+          ModalChangeReasonController: {
+            reasonId: 1,
+            date: today
+          }
+        });
+        makeController({ action: 'change' });
+      });
+
+      describe('normal behaviour', function () {
```

Comment:
could you please change to 'basic tests' since when you read 'normal behaviour' you automatically ask yourself 'so is there is abnormal behaviour somewhere?' and there is nothing abnormal in errors.